cluster/juju/layers/kubernetes-worker/reactive/kubernetes_worker.py

```diff
@@ -284,11 +293,12 @@ def render_init_scripts(api_servers):
     kubelet_opts = FlagManager('kubelet')
     # Declare to kubelet it needs to read from kubeconfig
     kubelet_opts.add('--require-kubeconfig', None)
-    kubelet_opts.add('--kubeconfig', '/srv/kubernetes/config')
+    kubelet_opts.add('--kubeconfig', kubeconfig_path)
+    kubelet_opts.add('--network-plugin', 'cni')
```

Comment:
This new flag is incompatible with software defined networking (which is totally fine) but we could put this line inside an if block when using CNI. I know that is not in the scope of this PR. Given your expertise: Would we be able to support both SDN and CNI with these charms?